example/test_problem/Hydro/CCSN/Input__Flag_Lohner

```diff
@@ -1,10 +1,10 @@
-# Level           Threshold              Filter              Soften        MinDensity
-      0                0.80                0.01                0.00              0.00
-      1                0.80                0.01                0.00              0.00
-      2                0.80                0.01                0.00              0.00
-      3                0.80                0.01                0.00              0.00
-      4                0.80                0.01                0.00              0.00
-      5                0.80                0.01                0.00              0.00
-      6                0.80                0.01                0.00              0.00
-      7                0.80                0.01                0.00              0.00
-      8                0.80                0.01                0.00              0.00
+# Level    Threshold_Refine  Threshold_Derefine            Filter              Soften        MinDensity
+      0                0.70                0.30              0.01                0.00              0.00        
+      1                0.70                0.30              0.01                0.00              0.00
+      2                0.70                0.30              0.01                0.00              0.00
+      3                0.70                0.30              0.01                0.00              0.00
+      4                0.70                0.30              0.01                0.00              0.00
+      5                0.70                0.30              0.01                0.00              0.00
+      6                0.70                0.30              0.01                0.00              0.00
+      7                0.70                0.30              0.01                0.00              0.00
+      8                0.70                0.30              0.01                0.00              0.00
```

Comment:
- Create a new file `Input__Flag_Lohner.CoreCollapse`, and keep the original one the same for other sub test problem (You still need to add `Threshold_Derefine` and set the value to 0.8)
- Why use 0.7 and 0.3 for refinement and derefinement, respectively? Any test on this setting?
- Remove the trailing spaces
- For the `Filter` and `MinDensity`, keep the column width to be 20. See, for example, `Riemann/Input__Flag_Lohner`

Reply:
I have added `Input__Flag_Lohner.*` and changed the threshold for core collapse to `0.8` and `0.4` as you tested.

Reply:
@Inhyeok-Song I reopen it since some questions are not resolved.
- [ ] Why use 0.7 and 0.3 for refinement and derefinement, respectively? Any test on this setting?
- [ ] Why do we still need this file given that there are already `Input__Flag_Lohner.*` for different tests?

Reply:
I removed `Input__Flag_Lohner`.